boot.py

```diff
@@ -4,92 +4,92 @@
 import sublime_plugin
 
 # Please keep this list sorted (Edit -> Sort Lines)
-from .plugin.code_actions import LspCodeActionsCommand
-from .plugin.code_actions import LspRefactorCommand
-from .plugin.code_actions import LspSourceActionCommand
-from .plugin.code_lens import LspCodeLensCommand
-from .plugin.code_lens import LspToggleCodeLensesCommand
-from .plugin.color import LspColorPresentationCommand
-from .plugin.completion import LspCommitCompletionWithOppositeInsertMode
-from .plugin.completion import LspResolveDocsCommand
-from .plugin.completion import LspSelectCompletionCommand
-from .plugin.configuration import LspDisableLanguageServerGloballyCommand
-from .plugin.configuration import LspDisableLanguageServerInProjectCommand
-from .plugin.configuration import LspEnableLanguageServerGloballyCommand
-from .plugin.configuration import LspEnableLanguageServerInProjectCommand
-from .plugin.core.collections import DottedDict
+from .plugin.code_actions import LspCodeActionsCommand  # noqa: F401
```

Comment:
I would prefer `__all__`.

Reply:
But `__all__` requires more knowledge about what should be exported by design (I don't know).

Reply:
The `boot` module only needs to expose all ST API classes, such as commands and event listeners as well as `plugin_loaded` and `plugin_unloaded` as it is the plugin-module being imported by ST.

Reply:
> The `boot` module only needs to expose all ST API classes, such as commands and event listeners as well as `plugin_loaded` and `plugin_unloaded` as it is the plugin-module being imported by ST.

I know this, but there are more. Otherwise, they don't have to be imported. At the same time, some of used imports, should be exposed. But I don't know.

Reply:
IMHO, actually every imported class with a noqa: F401 + the Listener and plugin_loaded/plugin_unloaded API functions.

Everything else is imported for being used in this module.

Reply:
Pretty sure some of [`1fcf88c` (#2459)](https://github.com/sublimelsp/LSP/pull/2459/commits/1fcf88c24f895719f2abaf044a3882691d15ab6a) are not intended to be exposed.

Reply:
FWIW,
this is not documented,

LSP-* packages should only import from the API that LSP exposes in `LSP.plugin` 
https://github.com/sublimelsp/LSP/blob/main/plugin/__init__.py

 or `LSP.plugin.core.protocol` or `LSP.plugin.core.types`.

If someone imported form a place that is not these three files,
than ideally the package needs to tweak the logic to import the function form `LSP.plugin`.

if the function doesn't exist in `LSP.plugin` than open a issue to add discus exposing that function.